Ports/chocolate-doom/package.sh

```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port='chocolate-doom'
-version='3.0.1'
+version='3.1.0'
+workdir="${port}-${port}-${version}"
 useconfigure='true'
-use_fresh_config_sub='true'
-config_sub_paths=('autotools/config.sub')
 files=(
-    "https://www.chocolate-doom.org/downloads/${version}/chocolate-doom-${version}.tar.gz#d435d6177423491d60be706da9f07d3ab4fabf3e077ec2a3fc216e394fcfc8c7"
+    "https://github.com/${port}/${port}/archive/refs/tags/${port}-${version}.tar.gz#f2c64843dcec312032b180c3b2f34b4cb26c4dcdaa7375a1601a3b1df11ef84d"
```

Comment:
Please continue using the tarballs from the website, that should take care of the autoconf issue.

Reply:
There is no 3.1.0 tarball on the website. You can confirm by looking at. https://www.chocolate-doom.org/downloads/